src/components/TileGrid/TileEdit.tsx

```diff
@@ -27,21 +62,39 @@ function TileEditMenu({ visible, setVisible, options }: ITileEditMenu) {
     <div
       className={cn(
         visible ? "flex" : "hidden",
-        "absolute top-0 right-0 z-10 flex-col text-center rounded bg-white drop-shadow-md"
+        "absolute z-10 flex-col rounded bg-white text-center drop-shadow-md"
       )}
+      style={{ borderRadius: "6px", paddingInline: "10px" }}
     >
       <form method="dialog">
-        {options.map((option) => (
-          <button
-            key={option.name}
-            className="w-full p-2 px-4 hover:bg-offer-white"
-            onClick={(e) => {
-              option.onClick(e);
-              setVisible(false);
-            }}
-          >
-            {option.name}
-          </button>
+        {options.map((option, index) => (
+          <>
+            <button
+              key={option.name}
+              className="w-full p-2 px-4 hover:bg-offer-white"
+              style={{
+                color: options[0] === option ? "#22555A" : "#EF6767",
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "space-between",
+              }}
+              onClick={(e) => {
+                option.onClick(e);
+                setVisible(false);
+              }}
+            >
+              <span
+                style={{
+                  marginRight: "20px",
+                  fontFamily: "Merriweather",
+                }}
+              >
+                {option.name}
+              </span>
+              {options[0] === option ? EditIcon() : TrashIcon()}
```

Comment:
It might make more sense if the client pass in the icon they would like to render. The way it is set up right now limits it to either an `EditIcon` or `TrashIcon`.